thriftbp/client_middlewares.go

```diff
@@ -14,6 +14,21 @@ import (
 	"github.com/reddit/baseplate.go/tracing"
 )
 
+// MonitorClientWrappedSlugSuffix is a suffix to be added to the service slug
+// arg of MonitorClient function, in order to distinguish from the spans that
+// are the raw client calls..
+//
+// The MonitorClient with this suffix will have span operation names like:
+//
+//     service.wrapped.endpointName
+//
+// Which groups all retries of the same client call together,
+// while the MonitorClient without this suffix will have span operation names
+// like:
+//
+//     service.endpointName
+const MonitorClientWrappedSlugSuffix = ".wrapped"
```

Comment:
Other options might be ".with-retry"?

Reply:
Or maybe "wrapped."/"with-retry." prefixes are better? Since on metrics we already have "clients." prefix before them.

Reply:
I think _maybe_ use `-wrapped` or `-with-retry`?  Then it's separated as:

```
client.foo-wrapped.fizz
client.foo.fizz
```

Reply:
That way it doesn't possibly conflict with either having a client named `wrapped` or a method named `wrapped`?

Reply:
What if we have another service called foo-wrapped :trollface: 

I think that makes sense. @konradreiche what do you think?

Reply:
:shakes-fist: that's true!!!

Reply:
I'd love to see this being explicit about the retry middleware, i.e. `wrapped-with-retry` if we're not too bothered by a longer name.

Reply:
`-with-retry` sounds great

Reply:
Changed to "-with-retry"